includes/class-sensei-course.php

```diff
@@ -3887,11 +3887,16 @@ public static function alter_redirect_url_after_enrolment( $url, $post ) {
 	 * Get average days to completion for all courses.
 	 *
 	 * @since 4.2.0
+	 *
+	 * @deprecated 4.5.0 use Sensei_Reports_Overview_List_Table_Courses::get_average_days_to_completion
+	 *
 	 * @access private
 	 *
 	 * @return float Average days to completion, rounded to the highest integer.
 	 */
 	public function get_average_days_to_completion() {
+		_deprecated_function( __METHOD__, '4.5.0', 'Sensei_Course::get_average_days_to_completion' );
```

Comment:
I think the third param should be the replacement method, in this case `Sensei_Reports_Overview_List_Table_Courses::get_average_days_to_completion`.

Reply:
Or maybe `Sensei_Reports_Overview_Service_Courses::get_average_days_to_completion` if you decide to move it there.